test/basic_any_cast_cv_failed.cpp

```diff
@@ -0,0 +1,14 @@
+// Copyright 2006 Alexander Nasonov.
+// Copyright Antony Polukhin, 2013-2019.
```

Comment:
Add your name on a new line in every file

Reply:
Done.